From: Jeff Mahoney <jeffm@suse.com>
Subject: [PATCH] acpi: unexport acpi_rs_match_vendor_resource

 acpi_rs_match_vendor_resource is a static function and according to Alexey,
 shouldn't be exported.

Cc: Alexey Starikovskiy <astarikovskiy@novell.com>
Signed-off-by: Jeff Mahoney <jeffm@suse.com>

---

 drivers/acpi/resources/rsxface.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/acpi/resources/rsxface.c~	2007-06-15 13:10:44.000000000 -0400
+++ b/drivers/acpi/resources/rsxface.c	2007-06-15 14:55:26.000000000 -0400
@@ -474,8 +474,6 @@
 	return (AE_CTRL_TERMINATE);
 }
 
-ACPI_EXPORT_SYMBOL(acpi_rs_match_vendor_resource)
-
 /*******************************************************************************
  *
  * FUNCTION:    acpi_walk_resources
